components/navbar.tsx

```diff
@@ -33,13 +45,23 @@ export default async function Navbar() {
         >
           Home
         </Link>
-        <a
-          href="#userTestimonials"
-          className="text-navbarSecondary hover:text-primary font-semibold transition md:block hidden dark:text-white"
-          style={{ scrollBehavior: "smooth" }}
-        >
-          Testimonial
-        </a>
+        {currentPath === "/leaderboard" ? (
+          <Link
+            href={"/#userTestimonials"}
+            className="text-navbarSecondary hover:text-primary font-semibold transition md:block hidden dark:text-white"
+            style={{ scrollBehavior: "smooth" }}
+          >
+            Testimonial
+          </Link>
+        ) : (
+          <a
+            href="#userTestimonials"
+            className="text-navbarSecondary hover:text-primary font-semibold transition md:block hidden dark:text-white"
+            style={{ scrollBehavior: "smooth" }}
+          >
+            Testimonial
+          </a>
+        )}
```

Comment:
Remove this

Reply:
If i remove  this will not work,

can you plz provide me more information ?

Reply:
@dhairyathedev

Reply:
The simple solution is to just change the href and don't need this pass checks.

Reply:
bro can you connect with me on Linkedin